hardhat.config.ts

```diff
@@ -210,8 +155,8 @@ const config: HardhatConfig = {
       tags: ['testnet', 'dev'],
       companionNetworks: {},
     },
-    [NETWORK.HARDHAT]: {
-      chainId: CHAINID.HARDHAT,
+    [hardhat.id]: {
```

Comment:
```suggestion
    [hardhat.name]: {
```

Reply:
hardhat network is fixed with 'hardhat'

Reply:
https://github.com/clober-dex/core/pull/9/commits/7188cf322ec8da2125bd07949c4786eecc43c877 @JhChoy  How about this?

Reply:
@JhChoy https://github.com/clober-dex/core/pull/9/commits/32200cc45ab7296b3922eabe5d20eaa18a04b330 Sure I fixed it